assets/scss/components/_archive-goals.scss

```diff
@@ -0,0 +1,9 @@
+.archive-goal-summary {
+  background-color: govuk-colour("light-grey");
+  padding: 25px;
+  margin-bottom: 25px;
+
+  h3 {
+    margin-bottom: 10px;
```

Comment:
Rather than hardcoding pixels I wonder if it's better to leverage the standard GDS styles on the component HTML elements themselves (which will be responsive with associated media queries etc where necessary)
EG. in the nunjucks, on the h3, how about `<h3 class="govuk-!-margin-bottom-2">` (or whatever the right number is)
That way you are using GDS styles, you are not hardcoding pixel dimensions, and then this sass becomes a "style overrides for things I cannot control with standard GDS styles"

Reply:
Sure, I'll try to change it but no promises, this is not my forte!  I followed the style of the DPS search box (the other grey box I could see in the system) and that uses px for margin.